samoa-instances/src/main/java/org/apache/samoa/instances/AvroLoader.java

```diff
@@ -254,12 +257,30 @@ protected InstanceInformation getHeader() {
         List<String> attributeLabels = attributeSchema.getEnumSymbols();
         attributes.add(new Attribute(field.name(), attributeLabels));
       }
-      else
+      else if (isNumeric(field))
```

Comment:
You are correct and hopefully agree that it is an improvement as it is now accepting of more data sets (including mine with has way too many string attributes - could add a pre-processing step/job before Samoa but thought this nice to have regardless).